pkg/assets/builder.go

```diff
@@ -95,7 +99,13 @@ func (a *AssetBuilder) remapImage(image string) (string, error) {
 	}
 
 	registryMirror := os.Getenv("DEV_KOPS_REGISTRY_MIRROR")
```

Comment:
We can dump the env var now that we have AssetRemapping in the API.. it was just to decouple the two PRs